web/src/navbar_alerts.ts

```diff
@@ -250,18 +249,37 @@ const bankruptcy_banner = (): AlertBanner => {
     };
 };
 
+const demo_organization_deadline_banner = (): AlertBanner => {
+    const days_remaining = get_demo_organization_deadline_days_remaining();
+    return {
+        process: "demo-organization-deadline",
+        intent: days_remaining <= 7 ? "danger" : "info",
+        label: new Handlebars.SafeString(
+            $t_html(
+                {
+                    defaultMessage:
+                        "This <demo_link>demo organization</demo_link> will be automatically deleted in {days_remaining} days, unless it's <convert_link>converted into a permanent organization</convert_link>.",
```

Comment:
These are better names than the old `z-general-link`, but do note that sort of change to strings require translators to re-translate the strings, which can be a lot of human work.

We also have a convention of using `<z-*link>` for these names. So `z-demo-link` and `z-convert-link` would be OK.

Reply:
I'll keep that in mind, Changed the link translation tags to `z-demo-link` and `z-convert-link`.